### PR TITLE
QPPSF-5230 - Refined Y3 Clinical Clusters

### PR DESCRIPTION
**Pull request by shadford**
Include any related JIRA issue keys in the PR title, eg `QPPA-1234 Add measures x, y, and z`.

#### Motivation for change

What is the problem being solved? What is the feature being added?

#### What is being changed

Detailed information about what is being changed and rationale for interesting decisions made

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [ ] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [ ] Documentation updated
* [ ] Unit tests added/passing
* [ ] Verified working locally

##### Associated JIRA tickets:
* https://jira.cms.gov/browse/QPPSF-5230

@harrisj @lamroger @sheldon-b @kyeah @shebryant @KyleApfel @NateTheGreatt @Jpec07 @bijujoseph

### CODE REVIEW

File: clinical-clusters/2019/clinical-clusters.json [hunk 1]
```diff
@@ -1012,37 +1157,39 @@
...
+    "specialtySets": [
       {
-        "name": "endoscopyAndPolypSurveillance",
+        "name": "radiationOncology",
         "measureIds": [
-          "185",
-          "320"
+          "102",
+          "143",
+          "144"
```
Comment (KyleApfel) [inline clinical-clusters/2019/clinical-clusters.json]: According to https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPPRACTICE&title=EMA+Logic+Year+3 it looks like `radiationOncology ` was supposed to be removed. Might be some context I don't know...

Comment (shadford) [reply clinical-clusters/2019/clinical-clusters.json]: the ema cluster builder reorders everything so git thinks it was updated, this one is actually  for:  Radiation Oncology* | Registry rather then Claims, if you scroll to the bottom of the speciality scenarios you can see that one for measures 102, 143, 144

Comment (shadford) [reply clinical-clusters/2019/clinical-clusters.json]: if you look for claims for 156 the whole measure is no longer in the list...

**Comment by shadford** [comment]
the version is bumped in #293 and will be deployed at the same time as this pr

**Pull request merged**